src/tests/topp/ProteinQuantifier_14_output2.txt

```diff
@@ -1,7 +1,7 @@
-# Peptide abundances computed from file '/ceph/ibmi/abi/projects/sachsenb/OpenMS/src/tests/topp/ProteinQuantifier_input.idXML'
+# Peptide abundances computed from file '/home/sachsenb/Development/OpenMS/src/tests/topp/ProteinQuantifier_input.idXML'
 # Parameters (relevant only): (none)
 # Files/samples associated with abundance values below: 1: 'file1.mzML', 2: 'file2.mzML'
-"peptide"	"protein"	"n_proteins"	"charge"	"abundance_1"	"abundance_2"	"fraction"
+"peptide"	"protein"	"n_proteins"	"charge"	"abundance_file1.mzML_1"	"abundance_file2.mzML_1"	"fraction"
```

Comment:
Do you think we should add the prefix in a bit more differentiable way?
Right now it is not very self-explanatory and might be confused as part of the filename (especially in the PQ_17_output.txt. Which brings me to my next question: Why do the column names sometimes have .mzml in the name and sometimes it is stripped away?

Reply:
I was thinking about a different separator that is less commonly used in filenames (e.g. | pipe) and to use chN to signal that it is a channel number.